Modified by Peter Michael Green for http 2.x package.

Description: avoid crate quickcheck
 Uses old release of quickcheck unavailable in Debian.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-04-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: http/tests/header_map_fuzz.rs
===================================================================
--- http.orig/tests/header_map_fuzz.rs
+++ http/tests/header_map_fuzz.rs
@@ -1,3 +1,4 @@
+/*
 use http::header::*;
 use http::*;
 
@@ -373,3 +374,4 @@ fn gen_string(g: &mut StdRng, min: usize
 
     String::from_utf8(bytes).unwrap()
 }
+*/
Index: http/Cargo.toml
===================================================================
--- http.orig/Cargo.toml
+++ http/Cargo.toml
@@ -44,12 +44,6 @@ version = "0.3"
 [dev-dependencies.indexmap]
 version = "<=1.8"
 
-[dev-dependencies.quickcheck]
-version = "0.9.0"
-
-[dev-dependencies.rand]
-version = "0.7.0"
-
 [dev-dependencies.seahash]
 version = "3.0.5"
 
